examples-generated/bedrock/v1beta1/modelinvocationloggingconfiguration.yaml

```
@@ -0,0 +1,70 @@
+apiVersion: bedrock.aws.upbound.io/v1beta1
+kind: ModelInvocationLoggingConfiguration
+metadata:
+  annotations:
+    meta.upbound.io/example-id: bedrock/v1beta1/modelinvocationloggingconfiguration
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    region: us-west-1
```

You should add `loggingConfig` block like the following and create the resource example under the `examples/bedrock/v1beta1/modelinvocationloggingconfiguration.yaml`

```suggestion
    region: us-west-1
    loggingConfig:
      embeddingDataDeliveryEnabled: true
      imageDataDeliveryEnabled: true
      s3Config:
        bucketName: example
```